src/gui_main.c

```diff
@@ -1007,6 +1008,26 @@ static GdkPixbuf* create_pixbuf(const gchar *filename)
     return pix;
 }
 
+static void gui_set_window_title(const char* title)
+{
+    gtk_window_set_title(GTK_WINDOW(MainWin), title);
+}
+
+static void gui_update_window_title(void)
+{
+    char* temp = "";
+    if (pref_get_show_path_in_title())
```

Comment:
Won't this leak memory if this branch or the one below with perf_get_show_version_in_title() is taken?

Initially, temp is a const char* to "; on the first call to g_strdump_printf, it allocates. Subsequent calls assign temp to the output of g_strdump_printf, and the memory allocated for it will be lost???

Reply:
My C skills are not that strong, so I'm not sure. I was patterning the code after line 995: static GdkPixbuf* create_pixbuf(const gchar *filename) which does what I think is exactly the same thing. The main difference being that I assign temp to "" first. Is the fix to just remove that? I could run a valgrind, but I'm super rusty on all that. Let me know what you think, or what fix you suggest.

Reply:
Can't answer on your comment about *Folder, so I will answer here. You have to launch with `-f`, that is what populates *Folder. To dynamically change it has not been implemented yet.